google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -513,6 +539,14 @@ private Value() {}
    */
   public abstract List<String> getStringArray();
 
+  /**
+   * Returns the value of an {@code ARRAY<JSON>}-typed instance. While the returned list itself will
+   * never be {@code null}, elements of that list may be null.
+   *
+   * @throws IllegalStateException if {@code isNull()} or the value is not of the expected type
+   */
+  public abstract List<String> getJsonArray();
```

Comment:
```suggestion
  public List<String> getJsonArray() {
    throw new UnsupportedOperationException("Not implemented");
  }
```

Reply:
We should provide a default implementation not to break any users who have their own implementation of the interface.